sdk/appconfiguration/app-configuration/review/app-configuration.api.md

```diff
@@ -98,29 +86,13 @@ export interface GetConfigurationSettingOptions extends RequestOptionsBase, Http
 
 // @public
 export interface GetConfigurationSettingResponse extends ConfigurationSetting, GetConfigurationHeaders, HttpResponseField<GetConfigurationHeaders> {
-}
-
-// @public
-export interface GetKeyValueHeaders {
-    eTag?: string;
-    lastModifiedHeader?: string;
-    syncToken?: string;
-}
-
-// @public
-export interface GetKeyValuesHeaders {
-    syncToken?: string;
-}
-
-// @public
-export interface GetRevisionsHeaders {
-    syncToken?: string;
+    statusCode: number;
 }
 
 // @public
 export interface HttpConditionalFields {
-    ifMatch?: string;
-    ifNoneMatch?: string;
+    onlyIfChanged?: boolean;
+    onlyIfUnchanged?: boolean;
 }
```

Comment:
what about `onlyIfMissing` / `onlyIfPresent`? Are these special Etags not supported by appconfig?

Reply:
onlyIfPresent doesn't exist in our current interface (conceptually it used to, as part of an 'update' method) but was scrapped early on in favor of just having add and set.

'add' implies 'onlyIfMissing' so it's not exposed as an option and set, which acts as an upsert, wouldn't use onlyIfPresent (or 'onlyIfMissing').